code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -25,6 +25,7 @@
 		log_game("Reagent explosion reaction occurred at [AREACOORD(T)]. Last Fingerprint: [lastkey ? lastkey : "N/A"]." )
 		var/datum/effect_system/reagents_explosion/e = new()
 		e.set_up(power , T, 0, 0)
+		SEND_SIGNAL(e, COMSIG_REAGENT_EXPLOSION, holder.my_atom)
```

Comment:
I think we should just set the explosion multiplier of the reagent_expolosion here, rather than using a kind of odd signal reaction.

Reply:
I don't know why i did it this way in the first place; done